src/mpyl/project.py

```diff
@@ -610,6 +610,9 @@ def from_config(values: dict, project_path: Path):
         )
 
 
+# root_dir is necessary because we load mpyl_stages.schema.yml dynamically
+# Follow-up question: why do we allow different repositories to define their own stages schema, but the project schema
```

Comment:
Because this was the first phase of making it all dynamic. You even approved (and liked) the pr/initiative from Sam iirc 😬 everything worked as is and we had higher prio things to work on, so it kinda got stuck I guess.